retis/src/core/events/bpf.rs

```diff
@@ -487,39 +484,20 @@ impl BpfEventsFactory {
     }
 }
 
-/// Size of msg for a single log event. Please keep in sync with its
-/// BPF counterpart.
-pub(super) const BPF_LOG_MAX: usize = 127;
-
-/// Max number of log events we can store at once in the shared map. Please keep in
-/// sync with its BPF counterpart.
-pub(super) const BPF_LOG_EVENTS_MAX: u32 = 32;
-
-event_byte_array!(BpfLogMsg, BPF_LOG_MAX);
-
-/// Log event. Please keep in sync with its BPF counterpart.
-#[derive(Default)]
-#[repr(C, packed)]
-pub(super) struct LogEvent {
-    level: u8,
-    msg: BpfLogMsg,
-}
-
-unsafe impl Plain for LogEvent {}
-
 /// Max number of events we can store at once in the shared map. Please keep in
 /// sync with its BPF counterpart.
 pub(super) const BPF_EVENTS_MAX: u32 = 8 * 1024;
 
 /// Size of the raw data buffer of a BPF event. Please keep synced with its BPF
 /// counterpart.
-pub(crate) const BPF_RAW_EVENT_DATA_SIZE: usize = 1024 - 2 /* remove the size field */;
+pub(crate) const BPF_RAW_EVENT_DATA_SIZE: usize = 1024 - 8 /* remove the size and pad fields */;
```

Comment:
Maybe we can use `mem::offset_of!` instead to improve maintenance?

Reply:
let's make this a binding and zero this maintenance :)